src/core/Notification/Notification.test.tsx

```diff
@@ -0,0 +1,141 @@
+        <div>Test notification</div>
+      </Notification>
+    );
+
+    it('has the given content', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container.firstChild).toHaveTextContent('Test notification');
+    });
+
+    it('should match snapshot', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('classnames', () => {
+    const customClassNotification = (
+      <Notification
+        closeText="Close"
+        className="custom-class"
+        labelText="Lorem ipsum dolor sit"
+      >
+        Test content
+      </Notification>
+    );
+
+    it('contains base classname', () => {
+      const { container } = render(customClassNotification);
+      expect(container.firstChild).toHaveClass('fi-notification');
+    });
+
+    it('contains custom classname', () => {
+      const { container } = render(customClassNotification);
+      expect(container.firstChild).toHaveClass('custom-class');
+    });
+  });
+  const NotificationWithButtonProps = (
+    <Notification
+      labelText="Lorem ipsum dolor sit"
+      closeText="Close"
+      closeButtonProps={{
+        'aria-labelledby': 'test-element',
+        disabled: true,
+        className: 'testClass',
+      }}
+    >
+      Testcontent
+    </Notification>
+  );
+  it('Button should contains custom classname', () => {
+    const { getByRole } = render(NotificationWithButtonProps);
+    expect(getByRole('button')).toHaveClass('testClass');
+  });
+  it('Button should have given props', () => {
+    const { getByRole } = render(NotificationWithButtonProps);
+    expect(getByRole('button')).toHaveAttribute(
+      'aria-labelledby',
+      'test-element',
+    );
+    expect(getByRole('button')).toHaveAttribute('disabled');
+    expect(getByRole('button')).toHaveTextContent('Close');
+  });
+
+  const NotificationWithDefaultAriaLiveMode = (
+    <Notification
+      id="testId"
+      closeText="Close"
+      ariaLiveMode="off"
+      labelText="Lorem ipsum dolor sit"
+    >
+      Testcontent
+    </Notification>
+  );
+
+  it('should have specified aria-live mode', () => {
+    const { container } = render(NotificationWithDefaultAriaLiveMode);
+    expect(container.querySelector('#testId')).toHaveClass(
+      'fi-notification_text-content-wrapper',
+    );
+    expect(container.querySelector('#testId')).toHaveAttribute(
+      'aria-live',
+      'off',
+    );
+  });
+  test('onClick event is called when clicked', () => {
+    const mockClick = jest.fn();
+    const { getByRole } = render(
+      <Notification
+        closeText="Close"
+        onClick={mockClick}
+        labelText="Lorem ipsum dolor sit"
+      >
+        Test content
+      </Notification>,
+    );
+    const button = getByRole('button');
+    fireEvent.click(button);
+    expect(mockClick).toHaveBeenCalledTimes(1);
+  });
```

Comment:
`onClick` prop to own block would be nice.

Reply:
Fixed.  Is now in own describe.

Reply:
Just nitpicking, but the describe block name does not match with the prop:
Currently it is `aria-live mode` when the prop name is actually  `ariaLiveMode`.

Reply:
Just a small typo in the describe block name: `aonClick`

Reply:
fixed typo.